src/apps/mesoscale-explorer/ui/entities.tsx

```diff
@@ -229,6 +230,165 @@ export class SelectionInfo extends PluginUIComponent<{}, { isDisabled: boolean }
     }
 }
 
+
+export function MesoViewportSnapshotDescription() {
+    let tSize = 14;
+    const plugin = React.useContext(PluginReactContext);
+    if (MesoscaleState.has(plugin)) {
+        const state = MesoscaleState.get(plugin);
+        tSize = state.textSizeDescription;
+    }
+    const [_, setV] = React.useState(0);
+    const [isShown, setIsShown] = useState(true);
+    const [textSize, setTextSize] = useState(tSize);
+    const toggleVisibility = () => {
+        setIsShown(!isShown);
+    };
+    const increaseTextSize = () => {
+        setTextSize(prevSize => prevSize + 2); // Increase the text size by 2px
+    };
+
+    const decreaseTextSize = () => {
+        setTextSize(prevSize => prevSize - 2); // Decrease the text size by 2px
+    };
```

Comment:
Has min/max text size?

Reply:
can you point me how to properly set the min/max here ?